hack/go-build.sh

```diff
@@ -5,5 +5,9 @@ set -e
 source hack/common.sh
 
 mkdir -p ${OUTDIR_BIN}
-go build -tags 'netgo osusergo' -ldflags="-s -w -X github.com/openshift/ocs-operator/controllers/defaults.IsUnsupportedCephVersionAllowed=${OCS_ALLOW_UNSUPPORTED_CEPH_VERSION}" -o ${OUTDIR_BIN}/ocs-operator ./main.go
-go build -tags 'netgo osusergo' -ldflags="-s -w -X github.com/openshift/ocs-operator/controllers/defaults.IsUnsupportedCephVersionAllowed=${OCS_ALLOW_UNSUPPORTED_CEPH_VERSION}" -o ${OUTDIR_BIN}/metrics-exporter ./metrics/main.go
+
+LDFLAGS="-s -w -X github.com/openshift/ocs-operator/controllers/defaults.IsUnsupportedCephVersionAllowed=${OCS_ALLOW_UNSUPPORTED_CEPH_VERSION} \
+	-X github.com/openshift/ocs-operator/version.Version=${CSV_VERSION}"
```

Comment:
Operator binary is built before building the CSV. So, we can not expect updated `CSV_VERSION` to be always available.
Instead add a new variable `VERSION` and default it to `CSV_VERSION`. That way I can choose to pass operator version explicitly, or default to CSV  (default) version or pass different CSV version to operator binary.

Reply:
PTAL, thanks!